ui/src/song/SongListActions.js

```diff
@@ -20,6 +22,7 @@ export const SongListActions = ({
   ids,
   ...rest
 }) => {
+  const isSmall = useMediaQuery((theme) => theme.breakpoints.up('sm'))
```

Comment:
This `isSmall` happens in a lot of places, see my comment about it being "inverted". It should be renamed or changed in all places